8vim/src/main/res/layout/touchtrail_settings_activity.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This activity should also have a preview box to view a sample trail as the user changes color.
Currently there is no feedback for the user.